hotdoc/extensions/gi/utils.py

```diff
@@ -1,10 +1,12 @@
 import os
 from collections import namedtuple
 import pathlib
+import traceback
 
 from backports.entry_points_selectable import entry_points
 
 from hotdoc.core.links import Link
+from hotdoc.utils.loggable import info, debug
```

Comment:
These are added simply because they're used in the `get_language_classes()` function later on, but would have been undefined if they'd actually been called.